partcad/tests/unit/test_git_clone_retries.py

```
@@ -83,15 +84,25 @@
+def git_config() -> callable:
+    def _git_config(key: str, /, *args, **kwargs) -> dict | str:
+        temp = defaultdict(dict, {
+            "git.clone.retry.max": test_git_retry_config["max"],
+            "git.clone.retry.patience": test_git_retry_config["patience"],
+        })
+        return temp[key]
+    return _git_config
+
 @pytest.mark.parametrize("git_error", fake_git_errors)
-def test_project_import_git_clone_retry_failure(git_error: GitCommandError):
+def test_project_import_git_clone_retry_failure(git_error: GitCommandError, git_config):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_project_import_git_clone_retry_failure`

Add return type annotation: `None`

(ANN201)